en_ewt-ud-train.conllu

```diff
@@ -43487,7 +43487,7 @@
 3	Leon	Leon	PROPN	NNP	Number=Sing	1	conj	1:conj:and|5:nsubj|6:nsubj:xsubj	_
 4	have	have	AUX	VBP	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	5	aux	5:aux	_
 5	proposed	propose	VERB	VBN	Tense=Past|VerbForm=Part	0	root	0:root	_
-6	xferring	xferr	VERB	VBG	VerbForm=Ger	5	xcomp	5:xcomp	_
+6	xferring	xfer	VERB	VBG	VerbForm=Ger	5	xcomp	5:xcomp	_
```

Comment:
Should this be marked as an abbreviation of "transfer"? lemma=transfer, Abbr=Yes, CorrectForm=transferring

Reply:
That makes sense.  Just made that update